src/Config.hs

```diff
@@ -159,7 +162,7 @@ data Package = Package {
 , packageLicense :: Maybe String
 , packageLicenseFile :: Maybe FilePath
 , packageExtraSourceFiles :: [FilePath]
-, packageSourceRepository :: Maybe String
+, packageSourceRepository :: Maybe (String, Maybe String)
```

Comment:
I think at this point I'm also in favor of adding a proper `SourceRepository` type (kudos to @jkarni).